### ISSUE TITLE
(feature) copy files to root of repository before pusing

### ISSUE THREAD

**Issue by fwolfst** [yegor256/jekyll-github-deploy#32]
I figured that for github "user" pages on github.io only the master branch can be used to serve the (via jgd: static) content.

I was very happy to see that with `jgd -b master -r source` I can still use jgd such that user.github.io will show me the rendered version.

However, now when browsing the repo on github, people do not see the README.md anymore (in which I would also state that I use jgd, and how). Thus it would be cool if jgd would have a `--copy-readme` (or `--copy-file README.md`) flag to copy the README.md over before pushing to the master branch.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by fwolfst** [comment]
I can draft a small nasty PR if you wish, basically guarding the `rm README` thing in the bash script.

**Comment by yegor256** [comment]
@fwolfst isn't is copying the file by default? I don't understand the problem...

**Comment by fwolfst** [comment]
@yegor256 yes its copied but then deleted: https://github.com/yegor256/jekyll-github-deploy/blob/587c67d481071a739c6681dff2ab419d0fd80db9/bash/deploy.sh#L55 . My PR adds an option to not delete the `README.md` .

**Comment by fwolfst** [comment]
In that sense, from a developer-perspective the flag shouldnt be called `copy-readme` but rather `do-not-delete-readme`, but from a users perspective that doesnt make much sense. Maybe `--keep-readme` ?
